### PR TITLE
[Add-ins] (Shared Runtime) Fixing Runtimes reference issue

### PR DESCRIPTION
**Pull request by mscharlock**

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[995d9f0](https://github.com/OfficeDev/office-js-docs-pr/commits/995d9f08615c598515206a291838cc9e0350e325)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[_dependentPackages/CommonPlugins/tools/JoinTOC.ps1](https://github.com/OfficeDev/office-js-docs-pr/blob/michellefixruntime/_dependentPackages/CommonPlugins/tools/JoinTOC.ps1) | :x:Error | | [Details](#user-content-402d9d56-583a0795)
[.openpublishing.publish.config.json](https://github.com/OfficeDev/office-js-docs-pr/blob/michellefixruntime/.openpublishing.publish.config.json) | :x:Error | | [Details](#user-content-d902ab53-583a0795)
[]() | :x:Error | | [Details](#user-content-583a0795)

<a id="402d9d56-583a0795"></a>
### [_dependentPackages/CommonPlugins/tools/JoinTOC.ps1](https://github.com/OfficeDev/office-js-docs-pr/blob/michellefixruntime/_dependentPackages/CommonPlugins/tools/JoinTOC.ps1)
 - **Line 82**: **[Error]** `````Error happened when executing plug-in script _dependentPackages/CommonPlugins/tools/JoinTOC.ps1 at step docset_prebuild. Please check your .openpublishing.publish.config.json?on the configuration of the plug-ins or ask for help?in Docs Support teams channel. Exit code: -1073741502`````
 - **Line 217**: **[Error]** `````Error happened when executing plug-in script _dependentPackages/CommonPlugins/tools/JoinTOC.ps1 at step docset_prebuild. Please check your .openpublishing.publish.config.json?on the configuration of the plug-ins or ask for help?in Docs Support teams channel. Please also include the build report and the following error code for our team to troubleshoot: 1`````

<a id="d902ab53-583a0795"></a>
### [.openpublishing.publish.config.json](https://github.com/OfficeDev/office-js-docs-pr/blob/michellefixruntime/.openpublishing.publish.config.json)
 - **Line 80**: **[Error]** `````Failed to execute customized script at step docset_prebuild in docset office-js-docs-pr. Please check your .openpublishing.publish.config.json?on the configuration of the plug-ins for docset office-js-docs-pr step docset_prebuild or ask for help?in Docs Support teams channel. Please also include the build report and the following error code for our team to troubleshoot: 1`````

<a id="583a0795"></a>
###
 - **Line 138**: **[Error]** `````W:\ij4a-s\.optemp\packages\opbuild.scripts.1.84.4\tools\opbuild\docset.prebuild.ps1 failed to execute`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C13%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202002132040277605-1595%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=4dOsTeVekpVLDCF1yNH8%2FSMva2hyEXPmSsftKhtK6So%3D&st=2020-02-13T20%3A36%3A19Z&se=2020-03-15T20%3A41%3A19Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[995d9f0](https://github.com/OfficeDev/office-js-docs-pr/commits/995d9f08615c598515206a291838cc9e0350e325)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/manifest/runtimes.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michellefixruntime/docs/reference/manifest/runtimes.md) | :white_check_mark:Succeeded | |
[docs/tutorials/share-data-and-events-between-custom-functions-and-the-task-pane-tutorial.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michellefixruntime/docs/tutorials/share-data-and-events-between-custom-functions-and-the-task-pane-tutorial.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C14%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202002141746366167-1595%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=q8sJs%2FqPAdKDTs4lpzyazmetZTLlf8uDx9GlEYDJItI%3D&st=2020-02-14T17%3A43%3A27Z&se=2020-03-16T17%3A48%3A27Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/reference/manifest/runtimes.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 description: ''
-ms.date: 01/24/2020
+ms.date: 02/13/2020
 localization_priority: Normal
 ---
 
 # Runtimes element
 
-This feature is in preview. Specifies the runtime of your add-in and allows custom functions and the task pane to share global data and make function calls into each other. Should follow the `<Host>` element in your manifest file.
+This feature is in preview. Specifies the runtime of your add-in and allows custom functions and the task pane to share global data and make function calls into each other. Child of the `<Host>` element in your manifest file.
```
Comment (ElizabethSamuel-MSFT) [inline docs/reference/manifest/runtimes.md]: Suggestion: Highlight that this (and other manifest elements) is in preview by using an alert (important or note). [Example](https://github.com/OfficeDev/office-js-docs-pr/commit/26754a60a94344b6f7f5e9834ebad34323ebb789#diff-fcf8cca2ad6b7b1c1a8e7725ded7fbd7)

Comment (ElizabethSamuel-MSFT) [inline docs/reference/manifest/runtimes.md]: Would it be useful to add a "Contained in" section that lists the Host element? [Example](https://docs.microsoft.com/en-us/office/dev/add-ins/reference/manifest/appdomains)

File: docs/tutorials/share-data-and-events-between-custom-functions-and-the-task-pane-tutorial.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-ms.date: 11/04/2019
+ms.date: 02/13/2019
```
Comment (ElizabethSamuel-MSFT) [inline docs/tutorials/share-data-and-events-between-custom-functions-and-the-task-pane-tutorial.md]:
```suggestion
ms.date: 02/13/2020
```

File: docs/tutorials/share-data-and-events-between-custom-functions-and-the-task-pane-tutorial.md [hunk 3]
```diff
@@ -49,9 +49,9 @@ After you complete the wizard, the generator creates the project and installs su
     <VersionOverrides xmlns="http://schemas.microsoft.com/office/taskpaneappversionoverrides" xsi:type="VersionOverridesV1_0">
       <Hosts>
         <Host xsi:type="Workbook">
-        <Runtimes>
-          <Runtime resid="TaskPaneAndCustomFunction.Url" lifetime="long" />
-        </Runtimes>
+            <Runtimes>
+                <Runtime resid="TaskPaneAndCustomFunction.Url" lifetime="long" />
+            </Runtimes>
         <AllFormFactors>
```
Comment (ElizabethSamuel-MSFT) [inline docs/tutorials/share-data-and-events-between-custom-functions-and-the-task-pane-tutorial.md]: It seems like AllFormFactors should also be indented further?
```suggestion
            <AllFormFactors>
```

**Comment by o365devx** [comment]
Docs Build status updates of commit _[a5fd949](https://github.com/OfficeDev/office-js-docs-pr/commits/a5fd9493ea7989ee5e73ff1adf2ee1c9d3e09bf2)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/manifest/runtime.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michellefixruntime/docs/reference/manifest/runtime.md) | :white_check_mark:Succeeded | |
[docs/reference/manifest/runtimes.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michellefixruntime/docs/reference/manifest/runtimes.md) | :white_check_mark:Succeeded | |
[docs/tutorials/share-data-and-events-between-custom-functions-and-the-task-pane-tutorial.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michellefixruntime/docs/tutorials/share-data-and-events-between-custom-functions-and-the-task-pane-tutorial.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C14%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202002142124254782-1595%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=KMAMNIdMgHNe%2BvYEM7%2B4QqvzIcSS2QpUpnfXX3pHeAo%3D&st=2020-02-14T21%3A20%3A36Z&se=2020-03-16T21%3A25%3A36Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/reference/manifest/runtime.md [hunk 4]
```diff
@@ -1,16 +1,19 @@
...
 
 # Runtime element
 
-This feature is in preview. Child element of the [`<Runtimes>`](runtimes.md) element. This element facilitates sharing of global data and function calls between Excel custom functions and the task pane of your add-in.
+Child element of the [`<Runtimes>`](runtimes.md) element. This element facilitates sharing of global data and function calls between Excel custom functions and the task pane of your add-in.
 
 **Add-in type:** Task pane
 
+> [!IMPORTANT]
+> Shared runtime is currently in preview and are only available on Excel on Windows. To try the preview features, you will need to join Office Insider.
```
Comment (ElizabethSamuel-MSFT) [inline docs/reference/manifest/runtime.md]: Could we also include a link to the Office Insider program?

```suggestion
> Shared runtime is currently in preview and only available in Excel on Windows. To try the preview features, you will need to join Office Insider.
```

**Comment by o365devx** [comment]
Docs Build status updates of commit _[0922c43](https://github.com/OfficeDev/office-js-docs-pr/commits/0922c43c4dbff78249fc68ff8071db8e1113c459)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/manifest/runtime.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michellefixruntime/docs/reference/manifest/runtime.md) | :white_check_mark:Succeeded | |
[docs/reference/manifest/runtimes.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michellefixruntime/docs/reference/manifest/runtimes.md) | :white_check_mark:Succeeded | |
[docs/tutorials/share-data-and-events-between-custom-functions-and-the-task-pane-tutorial.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michellefixruntime/docs/tutorials/share-data-and-events-between-custom-functions-and-the-task-pane-tutorial.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C18%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202002181849095783-1595%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=BnmQ7cmpZIYzHJEXUlptkXcE28E1Y9atBK6ivn8kdOo%3D&st=2020-02-18T18%3A45%3A24Z&se=2020-03-20T18%3A50%3A24Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[e55087d](https://github.com/OfficeDev/office-js-docs-pr/commits/e55087d9a776d084bcf0fb6d543567e3630bd541)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/manifest/runtime.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michellefixruntime/docs/reference/manifest/runtime.md) | :white_check_mark:Succeeded | |
[docs/reference/manifest/runtimes.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michellefixruntime/docs/reference/manifest/runtimes.md) | :white_check_mark:Succeeded | |
[docs/tutorials/share-data-and-events-between-custom-functions-and-the-task-pane-tutorial.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michellefixruntime/docs/tutorials/share-data-and-events-between-custom-functions-and-the-task-pane-tutorial.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C21%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202002211959241172-1595%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=DJA9%2FbvqGOl32EuMlh%2BUeB3x3xdOmwKq3tJvLUWeCdY%3D&st=2020-02-21T19%3A55%3A35Z&se=2020-03-23T20%3A00%3A35Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**